### PR TITLE
add Teller and Assemble to the footer and update translations

### PR DESCRIPTION
**Pull request by jinhojang6**
add Teller and Assemble to the footer and update all the language files.

Screen Shot 2020-01-23 at 1 24 00 AM

The order of the Status Network projects is
- Products
- Dev Tools
- Infra

(Suggested by Jonny)

### CODE REVIEW

File: themes/navy/layout/partial/footer.ejs [hunk 1]
```diff
@@ -64,6 +64,8 @@
                         <ul>
                             <li><a href="https://keycard.tech/" target="_blank"><%- __('site.footer.status-network.keycard') %></a></li>
                             <li><a href="https://dap.ps/" target="_blank"><%- __('site.footer.status-network.dap-ps') %></a></li>
+                            <li><a href="https://teller.exchange/#/" target="_blank"><%- __('site.footer.status-network.teller') %></a></li>
+                            <li><a href="https://assemble.fund/#/" target="_blank"><%- __('site.footer.status-network.assemble') %></a></li>
```
Comment (jakubgs) [inline themes/navy/layout/partial/footer.ejs]: You shouldn't include those `#/` in URLs.

Comment (jinhojang6) [reply themes/navy/layout/partial/footer.ejs]: Thanks, will fix those soon

**Comment by jinhojang6** [comment]
@jakubgs I removed the #s. Thank you for the feedback

**Comment by jakubgs** [comment]
Lol, unsigned again. You seriously need to edit your `.gitconfig` with:
```gitconfig
[user]
    email = jinho@status.im
    name = Jinho Jang
    signingkey = YOUR_KEY_ID
[alias]
    c   = commit --verbose --no-verify --gpg-sign --signoff
    m   = commit --verbose --no-verify --gpg-sign --signoff --amend
```
The aliases will help you never forget to sign your commits. Then you can just use `git c` or `git m`.

**Comment by jinhojang6** [comment]
@jakubgs I've got a bad habit forgetting the sign using -S sometimes with my new mac. Sorry and I've signed it!

Will configure the .gitconfig from now on

**Pull request merged**